protocol/storage/prover.go

```diff
@@ -37,7 +37,7 @@ type ProofReader interface {
 type ProofCalculator interface {
 	// CalculatePoSt computes a proof-of-spacetime for a list of sector ids and matching seeds.
 	// It returns the Snark Proof for the PoSt and a list of sector ids that failed.
-	CalculatePoSt(ctx context.Context, sortedCommRs proofs.SortedCommRs, seed types.PoStChallengeSeed) ([]types.PoStProof, []uint64, error)
+	CalculatePoSt(ctx context.Context, sortedCommRs go_sectorbuilder.SortedSectorInfo, seed types.PoStChallengeSeed) (types.PoStProof, error)
```

Comment:
nit: here and elsewhere the name `sortedCommRs` could be updated to the new thing for heightened clarity.

Reply:
done!